### PR TITLE
Added DocStrings for Camera type objects, and updated DocStrings for Scene type objects.

### PR DESCRIPTION
**Pull request by Aathish04**
DocStrings for viable methods of Camera, MovingCamera, ThreeDCamera, MappingCamera and MultiCamera have been added.

Some methods were not mentioned outside of their definitions, and DocStrings for these could be made more detailed.

The previous DocStrings written for classes in manimlib/scene have been updated to follow the numpy documentation format.

### CODE REVIEW

File: manimlib/camera/three_d_camera.py [hunk 1]
```diff
@@ -218,15 +365,45 @@ def get_static_center_func(mobject):
                 self.fixed_orientation_mobjects[submob] = func
 
     def add_fixed_in_frame_mobjects(self, *mobjects):
+        """This method allows the mobject to have a fixed position,
+        even when the camera moves around.
+        E.G If it was passed through this method, at the top of the frame, it
```
Comment (eulertour) [inline manimlib/camera/three_d_camera.py:370]: nit: e.g. should have a . after the g and the two letters needn't both be capitalized.

File: manimlib/camera/camera.py [hunk 2]
```diff
@@ -26,6 +26,19 @@
...
 
 class Camera(object):
+    """
+    Base Camera class.
+    This is the object which takes care of what exactly is displayed
+    on screen at any given moment.
+
+    Some important CONFIG values and local variables to note are:
+
+    self.background_image : str, optional
```
Comment (PgBiel) [inline manimlib/camera/camera.py:36]: Wrong format for attributes. (Note: with dataclasses we will be able to deal with this more appropriately.)
For now...

```py
"""(...)
    Attributes
    ----------
        background_image : str, optional
            The path to an image that should be the background image.
            If not set, the background is filled with `self.background_color`
    
        pixel_height : int
            (Text here???)
"""
```

File: manimlib/camera/camera.py [hunk 3]
```diff
@@ -51,6 +64,15 @@ class Camera(object):
     }
 
     def __init__(self, background=None, **kwargs):
+        """Initialises the Camera.
```
Comment (PgBiel) [inline manimlib/camera/camera.py:67]: Init documentation goes into the class docstring, using "Parameters". (Just move the Parameters section to the class docstring.)

File: manimlib/camera/camera.py [hunk 4]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
         return copy.copy(self)
 
     def reset_pixel_shape(self, new_height, new_width):
+        """This method resets the height and width
+        of a single pixel to the passed new_heigh and new_width.
+
+        Parameters
+        ----------
+        new_height : int, float
```
Comment (PgBiel) [inline manimlib/camera/camera.py:96]: 1. Use Union[a, b] for multiple type possibilities.
2. Use ``:class:`int`, :class:`float` ``

File: manimlib/camera/camera.py [hunk 5]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
...
+            The new height of the entire scene in pixels
+        new_width : int, float
```
Comment (PgBiel) [inline manimlib/camera/camera.py:98]: ^

File: manimlib/camera/camera.py [hunk 6]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
...
         self.resize_frame_shape()
         self.reset()
 
     def get_pixel_height(self):
+        """Returns the height of the scene in
+        pixel at that moment.
+
+        Returns
+        -------
+        int
```
Comment (PgBiel) [inline manimlib/camera/camera.py:113]: `` :class:`int` ``

File: manimlib/camera/camera.py [hunk 7]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
...
+        """
         return self.pixel_height
 
     def get_pixel_width(self):
+        """Returns the width of the scene in
+        pixels at that moment.
+
+        Returns
+        -------
+        int
```
Comment (PgBiel) [inline manimlib/camera/camera.py:124]: `` :class:`int` ``

File: manimlib/camera/camera.py [hunk 8]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
...
+        """
         return self.pixel_width
 
     def get_frame_height(self):
+        """Returns the height of the frame
+        in MUnits. Default is 8.0
+
+        Returns
+        -------
+        float
```
Comment (PgBiel) [inline manimlib/camera/camera.py:135]: `` :class:`float` ``

File: manimlib/camera/camera.py [hunk 9]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
...
+        """
         return self.frame_width
 
     def get_frame_center(self):
+        """Returns the absolute center of the frame as Cartesian
+        Coordinates with the unit MUnits.
+
+        Returns
+        -------
+        np.array
```
Comment (PgBiel) [inline manimlib/camera/camera.py:157]: `` :class:`array` ``  (we can link to external docs with sphinx)

File: manimlib/camera/camera.py [hunk 10]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
...
+            The array of x,y,z coordinates.
+        """
         return self.frame_center
 
     def set_frame_height(self, frame_height):
+        """Sets the frame height to the passed value.
+
+        Parameters
+        ----------
+        frame_height : int, float
```
Comment (PgBiel) [inline manimlib/camera/camera.py:167]: Union, `` :class:`int` `` and float too

File: manimlib/camera/camera.py [hunk 11]
```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
...
+        """
         self.frame_width = frame_width
 
     def set_frame_center(self, frame_center):
+        """Sets the center of the frame to the passed
+        cartesian coordinates.
+
+        Parameters
+        ----------
+        frame_center : np.array
```
Comment (PgBiel) [inline manimlib/camera/camera.py:188]: `` :class:`array` ``

File: manimlib/camera/camera.py [hunk 12]
```diff
@@ -102,6 +196,12 @@ def resize_frame_shape(self, fixed_dimension=0):
         of the pixels, where fixed_dimension determines
         whether frame_height or frame_width
         remains fixed while the other changes accordingly.
+
+        Parameters
+        ----------
+        fixed_dimension : int
```
Comment (PgBiel) [inline manimlib/camera/camera.py:202]: `` :class:`int` ``

File: manimlib/camera/camera.py [hunk 13]
```diff
@@ -116,6 +216,11 @@ def resize_frame_shape(self, fixed_dimension=0):
         self.set_frame_width(frame_width)
 
     def init_background(self):
+        """Initialize the background.
+        If self.background_image is the path of an image
```
Comment (PgBiel) [inline manimlib/camera/camera.py:220]: `` `self.background_image` `` - we need to use appropriate markup in docs

File: manimlib/camera/camera.py [hunk 14]
```diff
@@ -136,6 +241,20 @@ def init_background(self):
             self.background[:, :] = background_rgba
 
     def get_image(self, pixel_array=None):
+        """Returns an image from the passed
+        pixel array, or from the current frame
+        if the passed pixel array is none.
+
+        Parameters
+        ----------
+        pixel_array : np.array, list, tuple, optional
```
Comment (PgBiel) [inline manimlib/camera/camera.py:250]: use Union, and wrap each with ``:class:`...` ``

File: manimlib/camera/camera.py [hunk 15]
```diff
@@ -144,9 +263,32 @@ def get_image(self, pixel_array=None):
         )
 
     def get_pixel_array(self):
+        """Returns the pixel array
+        of the current frame.
+
+        Returns
+        -------
+        np.array
```
Comment (PgBiel) [inline manimlib/camera/camera.py:271]: `` :class:`array` ``

File: manimlib/camera/camera.py [hunk 16]
```diff
@@ -144,9 +263,32 @@ def get_image(self, pixel_array=None):
...
+        """
         return self.pixel_array
 
     def convert_pixel_array(self, pixel_array, convert_from_floats=False):
+        """Converts a pixel array from values that have floats in then
+        to proper RGB values.
+
+        Parameters
+        ----------
+        pixel_array : np.array, list, tuple
```
Comment (PgBiel) [inline manimlib/camera/camera.py:282]: Union, class

File: manimlib/camera/camera.py [hunk 17]
```diff
@@ -144,9 +263,32 @@ def get_image(self, pixel_array=None):
...
+            Pixel array to convert.
+        convert_from_floats : bool, optional
```
Comment (PgBiel) [inline manimlib/camera/camera.py:284]: `` :class:`bool` ``

File: manimlib/camera/camera.py [hunk 18]
```diff
@@ -144,9 +263,32 @@ def get_image(self, pixel_array=None):
...
+            Whether or not to convert float values to ints, by default False
+
+        Returns
+        -------
+        np.array
```
Comment (PgBiel) [inline manimlib/camera/camera.py:289]: class

File: manimlib/camera/camera.py [hunk 19]
```diff
@@ -157,6 +299,15 @@ def convert_pixel_array(self, pixel_array, convert_from_floats=False):
         return retval
 
     def set_pixel_array(self, pixel_array, convert_from_floats=False):
+        """Sets the pixel array of the camera to the passed pixel array.
+
+        Parameters
+        ----------
+        pixel_array : np.array, list, tuple
```
Comment (PgBiel) [inline manimlib/camera/camera.py:306]: Union, class

File: manimlib/camera/camera.py [hunk 20]
```diff
@@ -157,6 +299,15 @@ def convert_pixel_array(self, pixel_array, convert_from_floats=False):
...
+            The pixel array to convert and then set as the camera's pixel array.
+        convert_from_floats : bool, optional
```
Comment (PgBiel) [inline manimlib/camera/camera.py:308]: class

File: manimlib/camera/camera.py [hunk 21]
```diff
@@ -166,15 +317,35 @@ def set_pixel_array(self, pixel_array, convert_from_floats=False):
             self.pixel_array[:, :, :] = converted_array[:, :, :]
 
     def set_background(self, pixel_array, convert_from_floats=False):
+        """Sets the background to the passed pixel_array after converting
+        to valid RGB values.
+
+        Parameters
+        ----------
+        pixel_array : np.array, list, tuple
```
Comment (PgBiel) [inline manimlib/camera/camera.py:325]: Union, class

File: manimlib/camera/camera.py [hunk 22]
```diff
@@ -166,15 +317,35 @@ def set_pixel_array(self, pixel_array, convert_from_floats=False):
...
+            The pixel array to set the background to.
+        convert_from_floats : bool, optional
```
Comment (PgBiel) [inline manimlib/camera/camera.py:327]: class

File: manimlib/camera/camera.py [hunk 23]
```diff
@@ -189,10 +360,29 @@ def make_background_from_func(self, coords_to_colors_func):
...
         self.set_background(
             self.make_background_from_func(coords_to_colors_func))
 
     def reset(self):
+        """Resets the camera's pixel array
+        to that of the background
+
+        Returns
+        -------
+        Camera
```
Comment (PgBiel) [inline manimlib/camera/camera.py:383]: `` :class:`Camera` ``

**Pull request merged**